client/src/components/cityview/realm/villagers/VillagerComponent.tsx

```diff
@@ -15,8 +15,8 @@ type VillagerComponentProps = {
 
 export const VillagerComponent = ({ villager, setSelectedNpc }: VillagerComponentProps) => {
   const [showTravel, setShowTravel] = useState(false);
-  const { getNpcInfo } = useVillagerType();
-  const { getExtraButtons } = useVillagerButtons({ villager, setSelectedNpc, setShowTravel });
+  const { getVillagerTypeInfo } = villagerTypeInfo();
+  const { getVillagerButtons } = villagerButtons({ villager, setSelectedNpc, setShowTravel });
```

Comment:
Remove from a dictionary if there's only one key/value in the dict. Just return the functions directly

Reply:
Also, why return a function in the case of the buttons? Why not directly the value?

Reply:
And why not use directly a function in the case of the info, why do you use a function which returns a function?